tdmpc2/common/buffer.py

```diff
@@ -12,7 +12,7 @@ class Buffer():
 
 	def __init__(self, cfg):
 		self.cfg = cfg
-		self._device = torch.device('cuda')
```

Comment:
always safer to use ordinals - because `if self._device == tensor.device` could fail otherwise